hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/ApiaryGlueSync.java

```diff
@@ -51,13 +56,17 @@
 import com.amazonaws.services.glue.model.SerDeInfo;
 import com.amazonaws.services.glue.model.StorageDescriptor;
 import com.amazonaws.services.glue.model.TableInput;
+import com.amazonaws.services.glue.model.UpdateDatabaseRequest;
 import com.amazonaws.services.glue.model.UpdatePartitionRequest;
 import com.amazonaws.services.glue.model.UpdateTableRequest;
+import com.google.common.collect.ImmutableMap;
 
 public class ApiaryGlueSync extends MetaStoreEventListener {
 
   private static final Logger log = LoggerFactory.getLogger(ApiaryGlueSync.class);
 
+  private static final String CREATED_BY_GLUESYNC_KEY = "created-by";
```

Comment:
we can use eg-managed-by maybe

Reply:
this is open source so i don't think we can use the eg prefix, is `managed-by` better?

Reply:
doh, yeah managed-by than please

Reply:
updated